terraform_files/nutanix/variables_nutanix.tf

```diff
@@ -0,0 +1,132 @@
+//////
+// Nutanix variables
+//////
+
+variable "nutanix_endpoint" {
+  type        = string
+  description = "Endpoint for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_ENDPOINT environment variable"
+}
+
+variable "nutanix_username" {
+  type        = string
+  description = "Username for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_USERNAME environment variable"
+}
+
+variable "nutanix_password" {
+  type        = string
+  description = "Password for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_PASSWORD environment variable"
+}
+
+variable "nutanix_port" {
+  type        = number
+  description = "port for the Prism Elements or Prism Central instance. This can also be specified with the NUTANIX_PORT environment variable. Defaults to 9440"
+}
+
+variable "nutanix_cluster" {
+  type        = string
+  description = "Nutanix cluster name"
+}
+
+
+variable "nutanix_subnet" {
+  type        = string
+  description = ""
+}
+
+
+variable "nutanix_parent_folder" {
```

Comment:
This seems to be unused

Reply:
You right, I'll remove it.

Reply:
I thought that we might need it on our CI but now I think that we didn't.
If we will need it in the future I'll re-add it